typings/index.d.ts

```diff
@@ -362,7 +362,6 @@ declare module 'discord.js' {
     Events: {
```

Comment:
Not directly related to these changes, but why is `disconnect` part of `ClientEvents`? It's [not documented](https://discord.js.org/#/docs/main/stable/class/Client) and doesn't seem to be emitted? Which is weird because `ShardClientUtil.js` adds listeners for it (unless `ShardClientUtil#client` is a `Shard` object)

Reply:
1. The event was removed with 3f5161eb76b71ab8984e39ff39bf2937b82f5b43, and probably forgotten here.

2. I do not know how we want to handle that in `ShardClientUtil` now.
Using `Client#invalidated` instead could be an option. That however would require some internal refactoring as it currently is not emitted on the initial connecting process for some reason.